src/include/btree_cmp_inline.h

```diff
@@ -137,6 +137,93 @@ __wt_prefix_match(const WT_ITEM *prefix, const WT_ITEM *tree_item)
     return (__wt_lex_compare(prefix, tree_item, true));
 }
 
+/*
+ * __wt_btree_compare_bounds --
+ *     Return if the cursor key is within the bounded range. Calls the respective comparison
+ *     functions based on whether it's row or column store.
+ */
+static inline int
+__wt_btree_compare_bounds(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_COLLATOR *collator,
+  bool direction, bool *key_out_of_bounds)
+{
+    WT_BTREE *btree;
+
+    btree = S2BT(session);
+
+    if (btree->type == BTREE_ROW)
+        WT_RET(
+          __wt_row_compare_bounds(session, &cbt->iface, collator, direction, key_out_of_bounds));
```

Comment:
Make `__wt_row_compare and __wt_col_compare` args consistent with regards to cbt.